mempool/dydx_helpers.go

```diff
@@ -19,7 +19,8 @@ func IsClobOrderTransaction(
 		return false
 	}
 
-	if len(cosmosTx.Body.Messages) == 1 &&
+	if cosmosTx.Body != nil &&
```

Comment:
This was potentially a bug that could cause have caused panics.

Reply:
nice catch! in what case can the body be empty? is it possible to send an empty body tx?

Reply:
I am not sure if it's possible in practice, but this was happening in unit tests.